docker-compose.yml

```
@@ -11,8 +11,8 @@ services:
       - TESTVAR=${TESTVAR:-app}
     stdin_open: true
     tty: true
-    command: ["/bin/zsh"]
-    volumes:
+    command: ["/bin/zsh"] # default command
```

This will override the default command specified in the docker file
```suggestion
    command: ["/bin/zsh"] # override default command
```